Google BigQuery docs (#492)
objectiv/objectiv.io: docs/docs/tracking/collector/introduction.md

[modeling](../../modeling/index.mdx).

Our aim is to support all popular data stores. Currently supported are:

1. [PostgreSQL](./postgresql.md), out-of-the-box in a Docker container.
2. [Google BigQuery](./google-bigquery.md), using Snowplow as a pipeline and GCP.
3. [Amazon S3](./s3.md) (as an archive), using Snowplow  as a pipeline and AWS SQS/Kinesis.

:::info

If you want to test-drive Objectiv without having to set up a backend & data store, please apply for a 
[Launchpad](../../home/get-a-launchpad.md): a fully managed Objectiv back-end and data store that is free to 
use (limited slots).

:::

## First-party data

The Collector is self-hosted on your own domain, so no data is ever sent to any third-party, meaning:

* You have full control over your data.
* Tracking is compliant with privacy legislation such as GDPR, CCPA and PECR.
* Ad blockers can be avoided: first-party data tracking is usually not covered by ad blockers.

## Validation
The Collector validates any incoming Event against the 
[open analytics taxonomy](/taxonomy/reference/events/overview.md). If it fails, the Collector will respond 

> with an error, and store the Event in the `NOK` folder on disk.

Reviewer: This is a bit too on-disk-storage specific. How about:
```suggestion
with an error, and store the Event in the configured `NOK` (not-OK) location.

```
